schema/migrate-0.8.0-0.9.0.cql

```diff
@@ -144,3 +144,12 @@ CREATE TABLE caliopen.notification_ttl (
 // More dangerous operations
 ALTER TABLE caliopen.contact DROP deleted;
 ALTER TABLE caliopen.contact ADD deleted timestamp;
+
+// Configure notification TTLs
+INSERT INTO caliopen.notification_ttl (ttl_code, description, ttl_duration) values ('short-lived', 'short lieved'), 60);
```

Comment:
typo : `lieved` instead of `lived`, on that line and 2 lines below

Reply:
fixed in https://github.com/CaliOpen/deploy-alpha/pull/20/commits/65d3a9be056075bc20802417f0a8632123f66175